Makefile

```diff
@@ -1,12 +1,14 @@
 VERACRUZ_DOCKER_IMAGE ?= veracruz_image
 VERACRUZ_CONTAINER ?= veracruz
-VERACRUZ_ROOT ?= ..
+VERACRUZ_ROOT ?= ../veracruz
```

Comment:
Is there a reason we're still updating this repo, instead of using the `docker` directory in the `veracruz` project?